style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  display: flex;
+  background: url(background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
```

Comment:
- [ ] You didn't finish other TSE requirement. For stick design you should add same background image. It is almost same but you miss purple circle on bottom.

**Original :**
![image](https://user-images.githubusercontent.com/38635158/129234131-e16e4a51-9b39-45c0-8cc0-4dab13572166.png)
____
**Yours:**
![image](https://user-images.githubusercontent.com/38635158/129234267-d9476342-debf-451d-89ea-f88f29a9b1da.png)

Reply:
@ramiaquista You are right ı miss it because of scrolling.Sorry about that. You can request code review Next TSE will approve it.

Reply:
No problem, thank's for the advice for the next project challenges!